README.md

```diff
@@ -104,3 +103,20 @@ For version 1.x.x:
 
 For version 2.x.x:
  `Forte.EpiserverRedirects.Configuration.AddAutomaticRedirects` flag to `false`.
+
+
+Import redirects from CSV file
```

Comment:
Maybe worth adding that these Comma-Separated-Values should be separated with semicolons?

Reply:
Done, also the example CSV file is provided